templates/components/api-server/component.js

```diff
@@ -44,6 +44,12 @@ template.server = {
         'loopback/server/models',
         '../common/models',
         './models',
+      ],
+      mixins: [
```

Comment:
I am pondering whether "mixins" is a good key name, perhaps "mixinSources" would be a better name? (See also "sources" key used for model sources above.)